src/IntegratorFactory.php

```diff
@@ -148,14 +153,47 @@ protected function getConfig(): IntegratorConfig
     }
 
     /**
-     * @return \SprykerSdk\Integrator\Executor\ManifestExecutorInterface
+     * @return \SprykerSdk\Integrator\Executor\Module\ModuleManifestExecutorInterface
      */
-    public function creatManifestExecutor(): ManifestExecutorInterface
+    public function createModuleManifestExecutor(): ModuleManifestExecutorInterface
     {
-        return new ManifestExecutor(
+        return new ModuleManifestExecutor(
             $this->createIntegratorLockReader(),
-            $this->createRepositoryManifestReader(),
             $this->createIntegratorLockWriter(),
+            $this->createRepositoryManifestReader(),
+            $this->createManifestExecutor(),
+        );
+    }
+
+    /**
+     * @return \SprykerSdk\Integrator\Executor\ReleaseGroup\ReleaseGroupManifestExecutor
+     */
+    public function createReleaseGroupManifestExecutor(): ReleaseGroupManifestExecutor
+    {
+        return new ReleaseGroupManifestExecutor(
+            $this->createFileBucketManifestReader(),
+            $this->createBucketFileStorage(),
+            $this->createManifestExecutor(),
+            $this->createGitRepository(),
+        );
+    }
+
+    /**
+     * @param string|null $pathToRepository
+     *
+     * @return \SprykerSdk\Integrator\VersionControlSystem\GitRepository
+     */
+    public function createGitRepository(?string $pathToRepository = null): GitRepository
+    {
+        return new GitRepository($pathToRepository ?? (string)getcwd(), new CliRunner());
```

Comment:
pls use APPLICATION_ROOT_DIR instead of (string)getcwd()

Reply:
Thanks ++